overlord/state/change.go

```diff
@@ -477,6 +477,32 @@ func (c *Change) Tasks() []*Task {
 	return c.state.tasksIn(c.taskIDs)
 }
 
+// LaneTasks returns all tasks from given lanes the state change depends on.
+// All tasks are returned if any of the lanes is 0 (the default lane).
```

Comment:
the doc comment needs updating to remove the 2nd sentence.

Reply:
Done